scripts/retrain_parser.py

```diff
@@ -0,0 +1,150 @@
+import os
+import spacy
+
+from spacy.tokens import Doc
+from spacy.gold import GoldParse, GoldCorpus
+from conllu.parser import parse_line, DEFAULT_FIELDS
+from tqdm import tqdm
+import random
+from spacy.util import minibatch, compounding
+from spacy import util
+from timeit import default_timer as timer
+from spacy.cli.train import print_progress
+import argparse
+import json
+
+def train_parser(train_json_path: str,
```

Comment:
Add a test for this function? It would be relatively straightforward to check that it would run on a tiny test fixture or something.